app/viewmodels/checkAnswers/sections/firstTransporter/FirstTransporterHasVatSummary.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers.sections.firstTransporter
+
+import models.CheckMode
+import models.requests.DataRequest
+import models.sections.firstTransporter.FirstTransporterVatModel
+import pages.QuestionPage
+import pages.sections.firstTransporter.FirstTransporterVatPage
+import play.api.i18n.Messages
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object FirstTransporterHasVatSummary {
+
+  def row(showActionLinks: Boolean)(implicit request: DataRequest[_], messages: Messages): SummaryListRow = {
+
+    val namePage: QuestionPage[FirstTransporterVatModel] = FirstTransporterVatPage
+
+
+
+    SummaryListRowViewModel(
+      key = "firstTransporterVat.hasVat.checkYourAnswers.label",
+      value = {
+        val value = request.userAnswers.get(namePage).map { firstTransporterModel =>
+          firstTransporterModel.hasVat match {
+            case true => "Yes"
+            case _ => "No"
+          }
+        }
+        ValueViewModel(Text(value.getOrElse("Not provided")))
```

Comment:
I don't think not provided is a valid scenario, it's either yes or no (and it's mandatory)